events/listener.go

```diff
@@ -139,11 +145,15 @@ func (w *Worker) DoWork(rawEvent []byte, eventHandlers map[string]EventHandler,
 	if fn, ok := eventHandlers[event.Name]; ok {
 		err = fn(event, apiClient)
 		if err != nil {
-			log.Printf("Error processing event. Event name: %v. Event id: %v Resource id: %v. Error: %v",
-				event.Name, event.Id, event.ResourceId, err)
+			log.WithFields(log.Fields{
+				"Event Name":  event.Name,
+				"Event ID":    event.Id,
+				"Resource ID": event.ResourceId,
+				"Err":         err,
+			}).Warn("Error processing event")
 		}
 	} else {
-		log.Printf("No handler registered for event %v", event.Name)
+		log.Debug("No handler registered for event %v", event.Name)
```

Comment:
It's actually a pretty odd case that we'd hit this logic and it should show up in non-debug logs if it happens. I'm think this should be warn level.